readthedocs/api/v2/templates/restapi/footer.html

```diff
@@ -6,7 +6,7 @@
   <div class="rst-versions rst-badge" data-toggle="rst-versions">
     <span class="rst-current-version" data-toggle="rst-current-version">
       <span class="fa fa-book">&nbsp;</span>
-      v: {{ current_version }}
+      v: {{ version.explicit_name }}
```

Comment:
Looks like we are passing `version` and `current_version` so it doesn't get shallowed in the loop, I think we can make this `version` instead of only having the `verbose_name`.

https://github.com/readthedocs/readthedocs.org-ghsa-3v5m-qmm9-3c6c/blob/56100454427561fd225b0315ac9b259eac0d9da6/readthedocs/api/v2/views/footer_views.py#L171-L171

Reply:
I think it should be more or less how you have described now.